src/instructlab/model/backends/backends.py

```diff
@@ -202,7 +203,8 @@ def ensure_server(
     host="localhost",
     port=8000,
     queue=None,
-) -> Tuple[multiprocessing.Process, str]:
+    server_process_func=None,
```

Comment:
@n1hility @danmcp FYI I couldn't think of a less invasive way to refactor the code than to not set a type annotation for the `server_process_func`. Can you think of anything cleaner?

Reply:
we discussed options on chat, decided this cleanup can wait till later